site-src/faq.md

```diff
@@ -54,13 +54,33 @@
 * **Q: Which Kubernetes versions are supported?<br>**
   A: See our policy on [Supported Version](https://gateway-api.sigs.k8s.io/concepts/versioning/#supported-versions)
 
-* **Q: Is SSL Passthrough supported?**
+* **Q: Is SSL Passthrough supported?<br>**
   A: SSL Passthrough (wherein a Gateway routes traffic with the [Transport
   Layer Security (TLS)][tls] encryption _intact_ to a backend service instead of
   terminating it) is supported by [TLSRoutes][tlsroute]. See the
   [TLS Guide][tlsguide] for more details about passthrough and other TLS
   configurations.
 
+* **Q: What's the difference between Gateway API and an API Gateway?<br>**
+  A: An API Gateway is a general concept that describes anything that exposes
+  capabilities of a backend service, while providing extra capabilities for
+  traffic routing and manipulation, such as load balancing, request and response
+  transformation, and sometimes more advanced features like authentication and
+  authorization, rate limiting, and circuit breaking.
```

Comment:
Tiny nit: One extra newline here will ensure these paragraphs are separated in the rendered text, which will make this easier to read.

Reply:
I played around with this, having the extra newline as you suggest.
However, when viewing the preview in netlify I noticed odd formatting.
Here's a screenshot with newlines

![image](https://github.com/kubernetes-sigs/gateway-api/assets/878251/e1c8d93a-5d43-45de-98a7-a2933bf77ae3)

and without

![image](https://github.com/kubernetes-sigs/gateway-api/assets/878251/e6442870-5d6b-467d-9d8b-ebcd4d58bd93)

hence why I went without.
I'm not sure how best to resolve this, without maybe dropping to raw html?

Reply:
Sorted.
Looks like the indentation was off on a number of blocks.

<img width="780" alt="image" src="https://github.com/kubernetes-sigs/gateway-api/assets/878251/0375ba00-c002-4b2f-a6e0-0b3fe6516e0b">